### DISCUSSION TITLE
Sorting single select fields

### DISCUSSION THREAD

**Discussion by sschuberth** [community/community#6225] [Projects and Issues] [answered]
"Single select" fields seem to sort by order of the defined options (not by the alphabetical order of their names)), which is exactly what I'd expect. However, the corresponding UI is a bit misleading, IMO:

![image](https://user-images.githubusercontent.com/349154/136834213-fbd1a634-546f-49da-8a73-6ac644e8fa73.png)

The "(A...Z)" and "(Z...A)" parts should simply be stripped, IMO, to not give the wrong impression of alphabetical ordering.

**Comment by douglaslassance** [comment]
Seems like they updated the sorting recently (see #6046) and probably they did not catch up on labels.
Definitely should be addressed though.

**Comment by Jay-o-Way** [comment, marked as answer]
Is this still an issue?

**Reply by sschuberth** [reply]
Seem like not so as "A" / "Z" have been renamed to "first" / "last", see

![image](https://user-images.githubusercontent.com/349154/210391732-64f3ace1-26bd-4218-ba02-24e99edbbc22.png)
